src/app/modules/form/public-form/public-form.controller.ts

```diff
@@ -485,33 +486,46 @@ export const handleFormAuthRedirect = [
 
 /**
  * NOTE: This is exported only for testing
- * Logs user out of SP / CP By deleting cookie
+ * Logs user out of SP / CP / MyInfo / SGID by deleting cookie
  * @param authType type of authentication
  *
  * @returns 200 with success message when user logs out successfully
  * @returns 400 if authType is invalid
  */
 export const _handleSpcpLogout: ControllerHandler<
```

Comment:
Should we rename this function now that it handles all auth types? :X

Reply:
renamed, how about `handlePublicAuthLogout`?